dascore/io/core.py

```
@@ -691,7 +708,21 @@ def _updated_after(self, resource, timestamp):
         """Determine if the resource was updated after specified mtime."""
         if not timestamp:
             return True
-        return Path(resource).stat().st_mtime > timestamp
+        try:
+            path = (
+                coerce_to_upath(resource)
+                if not is_local_path(resource)
+                else Path(resource)
+            )
+            return path.stat().st_mtime > timestamp
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)